### PR TITLE
add chrono-node w/ npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8222 
@pvnr0082t , @x09326 Please help me review this pull request, thank you.
Manually add v0.1.9 and v1.1.0 on git repo because npm auto-update can't get them
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wanasit/chrono
 * Official website (optional, not the repository): https://wanasit.github.io/pages/chrono/
 * NPM package url (optional): https://www.npmjs.com/package/chrono-node
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 675
   - Count of watchers: 24
   - Count of forks: 105
 * NPM download stats (optional):
   - Downloads in the last day: 645
   - Downloads in the last week: 6950
   - Downloads in the last month: 25559

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by x09326** [comment]
@kennynaoh 
I think you need to modify the second commit.
You added the all old versions.
It is not same as your commit message.

**Comment by pvnr0082t** [comment]
@kennynaoh I suggest that you can just add v0.1.9/1.1.0 npm can't get next time. It will make review easier.

**Comment by kennynaoh** [comment]
@pvnr0082t , @x09326 
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
> Manually add v0.1.9 and v1.1.0 **on** git repo because npm auto-update can't get them

**from**, not **on**

File: ajax/libs/chrono-node/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "dependencies": {
+    "moment": "^2.10.3"
+  },
+  "author": "Wanasit Tanakitrungruang",
+  "npmName": "chrono-node",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "chrono.!(*json)"
```
Comment (PeterDaveHello) [inline ajax/libs/chrono-node/package.json]: This is not a `dist` folder, should better use whitelist not blacklist.

**Comment by PeterDaveHello** [comment]
@kennynaoh could you send a PR to list CDNJS in the readme?
https://github.com/wanasit/chrono#cdn

File: ajax/libs/chrono-node/package.json (hunk 1 above)
Comment (kennynaoh) [inline ajax/libs/chrono-node/package.json]: I will be careful next time, thanks!

**Comment by kennynaoh** [comment]
@PeterDaveHello , sure!

**Pull request merged**